models/swagger/search_params.go

```diff
@@ -0,0 +1,12 @@
+package swagger
+
+type PageParams struct {
```

Comment:
Why do we need this one? we can just use filter.Page and eventually extend it's description